src/models/adr-details.ts

```diff
@@ -36,8 +114,8 @@ export interface ApplicantDetails {
 }
 
 export interface AdditionalNotes {
-  number?: string[];
-  guidanceNotes?: string[];
+  number?: additionalNotesNumberEnum[];
```

Comment:
ADR documentation states that this shouldn't be an enum in the backend due to the historic data containing erroneous values

Reply:
Same as permittedDangerousGoodsEnum case